activities/codebase-stewardship/criteria-for-codebase-stewardship.md

```diff
@@ -14,3 +16,63 @@ Here are the criteria the staff of the Foundation for Public Code will use to he
 * Societal impact: what is the potential societal impact of the codebase becoming public code and being reused?
 * Story: what is the story-telling and communication value of this codebase becoming public code and being reused?  
+
+## Evaluation questions
+
+The current owners of potential codebases are asked to answer these questions to help us better understand their codebase.
+
+### Business case
+
+1. What's the current budget committed?
+
+### Co-develop-ability
+
+1. What percentage of codebase covered by tests?
+2. Can developers run the tests on their laptop?
+3. What are the buildtime and runtime dependencies?
+4. How many modules does the codebase have?
+5. How many lines of code does the codebase have?
+6. Is there a diagram and description of the system architecture?
+7. What portion of the APIs are documented?
+8. What portion of the code is commented/has inline documentation?
+9. What portion of the documentation exists in English?
+10. Are there any proprietary dependencies?
+11. Are there written engineering guidelines to inform architectural and code structure decisions?
+12. Are there explicit style guidelines for how code should look?
+
+### Community
+
+1. What portion of contributors are staff, contractors/freelancer, 3rd party procurement/vendors or volunteer?
+2. How many contributors have contributed in the last 6 months?
+3. What is the average number of commits per week in 2019?
+4. How many organizations contribute to this codebase?
+5. Which of these does your community have for discussion among users: email at a specific addres, email list, forum, bugtracking system, other (write in)?
+6. Which of these does your community have for discussion among developers: email at a specific addres, email list, forum, bugtracking system, other (write in)?
+7. Do you have contributor guidelines to help new people contribute?
+8. Does the community currently use English?
+9. What other languages does your community use?
+
+### Community potential
+
+1. Is there a 2 to 4 year roadmap?
+2. Is there a public backlog in English or one you can make public?
+3. How far through your current set of goals are you?
+4. How realistic vs ambitious are your goals in the next 6 months?
+5. Is there a public bugtracking system in English or one you can make public?
+6. Is the codebase being deployed by another organization?
+7. Has any other organization expressed an interest in deploying the code?
+
+### Code reusability
+
+1. How long would it take a team of 2 devs to extract to make the code configurable such that it could be deployed for a different org?
+2. What languages are used in your project?
+
+### Reuse potential
+
+1. What portion of the codebase is only understood by 1 or 2 people?
+2. Is the relevant policy bundled with the code?
+3. Can the relevant policy be bundled with the code in English?
+
+### Societal impact
```

Comment:
Sorry for the oversight, but here are headings without content, which should be removed.

Reply:
see #329